### DISCUSSION TITLE
Multiple miner monitoring

### DISCUSSION THREAD

**Discussion by G5OYY-uk** [BitMaker-hub/NerdMiner_v2#538] [Ideas]
Hi all just wondering if it would be possible to monitor several nerdminers on a single screen so like one actual nerd miner and that screen tondisplay the stats for each of the other miners in a kind of.scrolling format like the dongle display and maybe display the miner name just an idea as I have loads of esp 32 boards but it is a pain to keep loading web to ensure they are all still running if the screen was scrolling you would.see a abnormal.hash rate a lot.sooner

**Comment by sany3001** [comment]
I am using ESP32-2432S028R 2,8" that aggregates hash rate of all miners connected to the same mining address and shows the number of connected miners.

**Comment by G5OYY-uk** [comment]
Ooooo that may be the way forward have to  see if it will port over to the
s3

**Comment by sany3001** [comment]
Just buy, flash and configure. Link to buy is on Github project main page.

**Comment by G5OYY-uk** [comment]
I have 15 esp 32 s3 boards laying about so do not really want to buy another when i have boards and displays laying about
